examples/pxScene2d/src/initGL.js

```diff
@@ -42,22 +42,23 @@ var _ws = require('ws')
 var _http = require('http')
 var _https = require('https')
 var urlmain = require("url")
-const {promisify} = require('util')
-const readFileAsync = promisify(fs.readFile)
-const ArrayJoin = Function.call.bind(Array.prototype.join);
-const ArrayMap = Function.call.bind(Array.prototype.map);
+var {promisify} = require('util')
+var readFileAsync = promisify(fs.readFile)
+var ArrayJoin = Function.call.bind(Array.prototype.join);
+var ArrayMap = Function.call.bind(Array.prototype.map);
```

Comment:
Your PR has 2 initGL.js files, of which original initGL.js is unused, but you're still adding changes to it. Please remove obsolete files to avoid further confusion